src/frontends/onnx/frontend/src/op/bitwise_and.cpp

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+
+#include "op/bitwise_and.hpp"
+#include "default_opset.hpp"
+
+OPENVINO_SUPPRESS_DEPRECATED_START
+using namespace ov::op;
+
+namespace ngraph {
+	namespace onnx_import {
+		namespace op {
+			namespace set_1 {
```

Comment:
Hi, thanks for your contribution! Please, next time - do not add per-namespace tabs :)

Reply:
@rghvsh To fix the style issues, call `make clang_format_fix_all` from your `/bulid` directory.
It will align the code style automatically to satisfy the Code Style CI check.

Reply:
There is still some not fixed style issues, for example 2 spaces instead of 4 spaces tab.
You can see the required style changes generated by the style check on github in the "Files Changed" tab (https://github.com/openvinotoolkit/openvino/pull/21585/commits/376fce998463f66e7cc502219a37714eecfd4ce5). 
Anyway, I really recommend to adjust the style automatically by `make clang_format_fix_all` or at least check it by `make clang_format_check_all` locally before pushing.

Reply:
Had some problems with my system was not able to run the command fixed it now. Will format check all code locally before pushing in future.

Thank You for the help :)